src/test/fuzz/cluster_linearize.cpp

```diff
@@ -320,6 +351,7 @@ FUZZ_TARGET(clusterlin_chunking)
     try {
         reader >> Using<DepGraphFormatter>(depgraph);
     } catch (const std::ios_base::failure&) {}
+    MakeConnected(depgraph);
```

Comment:
Do we always want to make the graphs connected? Seems like a possible reduction in coverage?

Reply:
I'm not particularly bothered by it for the reasons you mentioned. I'll let others weigh in.